src/app/shared/components/footer/footer.component.html

```diff
@@ -16,11 +16,21 @@
     </svg>
     <span
       >by
-      <a class="text-slate-500" href="https://www.hochfrequenz.de/"
+      <a
+        class="text-slate-500"
+        href="https://www.hochfrequenz.de/"
+        target="_blank"
+        rel="noopener noreferrer"
```

Comment:
Was tut diese Zeile?

Reply:
"When you open a link in a new tab using target="_blank", the newly opened page can use JavaScript to access the window.opener object. This could allow malicious actions such as:

Redirecting the original page to a phishing site.
Stealing data from the original page.
The rel="noopener noreferrer" attribute prevents this by:

noopener: Ensures that the window.opener object is null, so the new tab cannot access or manipulate the original page.
noreferrer: Also prevents the browser from sending the Referer header, which contains the URL of the originating page."